www/front_src/src/Resources/Details/detailsAtoms.ts

```diff
@@ -71,17 +64,26 @@ export const setGraphTabParametersDerivedAtom = atom(
   },
 );
 
+export const selectedResourcesDetailsAtom = atomWithStorage<{
+  parentResourceId?: number;
+  parentResourceType?: string;
+  resourceId?: number;
+  resourcesDetailsEndpoint?: string;
+} | null>('resource_details', null);
+
 export const selectedResourceDetailsEndpointDerivedAtom = atom((get) => {
```

Comment:
This is not relevant anymore because it's provided from `selectedResourcesDetailsAtom`